From 2ec06b777c3802639aab1e72db3ad4b809c835ed Mon Sep 17 00:00:00 2001
From: terry <terry@szwesion.com>
Date: Wed, 30 Aug 2017 12:02:06 +0800
Subject: [PATCH 3/4] DVB: add gpio initiate for DVB (KVIM2)

---
 board/khadas/kvim2/kvim2.c | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/board/khadas/kvim2/kvim2.c b/board/khadas/kvim2/kvim2.c
index 67ef9aa..ff5dfc9 100644
--- a/board/khadas/kvim2/kvim2.c
+++ b/board/khadas/kvim2/kvim2.c
@@ -401,6 +401,17 @@ int board_init(void)
 	//set output 1
 	setbits_le32(PREG_PAD_GPIO0_O, (1 << 24));
 
+	//pull up GPIODV_11 for VTV Lock LED
+	//set output mode
+	clrbits_le32(PREG_PAD_GPIO0_EN_N, (1 << 11));
+	//set output 1
+	setbits_le32(PREG_PAD_GPIO0_O, (1 << 11));
+	//pull down GPIODV_20 for VTV Antenna Power (5V)
+	//set output mode
+	clrbits_le32(PREG_PAD_GPIO0_EN_N, (1 << 20));
+	//set output 0
+	clrbits_le32(PREG_PAD_GPIO0_O, (1 << 20));
+
 	/*Power on GPIOAO_2 for VCC_5V*/
 	clrbits_le32(P_AO_GPIO_O_EN_N, ((1<<2)|(1<<18)));
 #ifdef CONFIG_USB_XHCI_AMLOGIC_GXL
-- 
1.9.1

