From fc110efde47a06de00b62057975841ad82c94fa8 Mon Sep 17 00:00:00 2001
From: terry <terry@szwesion.com>
Date: Thu, 31 Aug 2017 15:10:25 +0800
Subject: [PATCH 4/4] DVB: add support for DVB IR wakeup (KVIM2)

---
 board/khadas/configs/kvim2.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/khadas/configs/kvim2.h b/board/khadas/configs/kvim2.h
index 1d1e7ac..c7a8fd6 100644
--- a/board/khadas/configs/kvim2.h
+++ b/board/khadas/configs/kvim2.h
@@ -76,7 +76,7 @@
 #define CONFIG_IR_REMOTE_POWER_UP_KEY_CNT 4
 #define CONFIG_IR_REMOTE_USE_PROTOCOL 0         // 0:nec  1:duokan  2:Toshiba 3:rca 4:rcmm
 #define CONFIG_IR_REMOTE_POWER_UP_KEY_VAL1 0XEB14FF00 //amlogic tv ir --- power
-#define CONFIG_IR_REMOTE_POWER_UP_KEY_VAL2 0Xffffffff //amlogic tv ir --- ch+
+#define CONFIG_IR_REMOTE_POWER_UP_KEY_VAL2 0XFF00FE01 //dvb ir  --- power
 #define CONFIG_IR_REMOTE_POWER_UP_KEY_VAL3 0xffffffff //amlogic tv ir --- ch-
 #define CONFIG_IR_REMOTE_POWER_UP_KEY_VAL4 0xBA45BD02
 
-- 
1.9.1

